mythtv/libs/libmythbase/mythcommandlineparser.cpp

```diff
@@ -80,6 +80,10 @@
 
 #define TERMWIDTH 79
 
+#ifdef _WIN32
+using std::cerr;
+using std::endl;
+#endif
```

Comment:
Please don't add this here. Just change line 2870 to read ```std::cerr << blah << std::endl```.

Reply:
Done